udf-catalogue.tex

```diff
@@ -401,8 +401,12 @@ \subsubsection{ivo\_epoch\_prop(ra, dec, parallax, pmra, pmdec,
 parameters, the proper motion in longitude is given for the tangential
 plane (``has ${\textrm cos}(\delta)$ applied'').
 
+Items that are NULL on input MUST be NULL (in VOTable arrays, NaN) on
+output, and when a single component of the proper motion is NULL, the
+other component must be NULL-ed out on output, too.  Similarly, when a
+NULL parallax is passed in, the RV must be NULL-ed out on output, too.
```

Comment:
Why does a NULL input parallax require a NULL output rv?  If the rv is OK on input but you don't happen to have a parallax for it, the same rv should be a reasonable value for the propagated version.  Am I missing something?